data/json/items/melee/bludgeons.json

```diff
@@ -871,6 +871,7 @@
     "longest_side": "90 cm",
     "to_hit": { "grip": "solid", "length": "short", "surface": "any", "balance": "clumsy" },
     "qualities": [ [ "HAMMER", 1 ] ],
+    "weapon_category": [ "MACES" ],
```

Comment:
```suggestion
```

Reply:
Given the length if anything it would be the same as a sledgehammer.  It's basically designed to be an awful sledge.

Reply:
Should I add a great hammers flag and iclude it in the basic great hammers thing?

Reply:
Yeah that would make more sense at least.